backend/app/routes/web/search.py

```diff
@@ -53,7 +53,7 @@ async def search_post(
                 cursor=cursor,
                 facets=request.facets,
             )
-        except TransportError as e:
+        except Exception as e:
```

Comment:
Too broad exception handling. If `TransportError` is not enough, let's add others